app/views/posts/show.html.erb

```diff
@@ -5,6 +5,12 @@
     <div class="post-stats">
       <span>Comments: <%= @post.comments.count %></span>
       <span>Likes: <%= @post.likes.count %></span>
+      <div class="like-button">
+      <%= form_with(url: user_post_likes_path(@author, @post), method: :post, local: true) do |form| %>
+        <%= form.hidden_field :author_id, value: current_user.id %>
+        <%= form.submit "Like" %>
+      <% end %>
+      </div>
     </div>
```

Comment:
- [ ] [OPTIONAL] Great job on creating the new post and new comments forms and they are as expected however I would suggest creating a new post button on `users/:id/posts` path. It's not very obvious or logical for user to have at least one post to  access the create new posts form.

Reply:
Thanks, I really appreciate your feedback. I'll apply your suggestion. 😊